app/steps/buildProxyReq.js

```diff
@@ -9,7 +9,8 @@ function buildProxyReq(Container) {
   var options = Container.options;
   var host = Container.proxy.host;
 
-  var parseBody = (!options.parseReqBody) ? Promise.resolve(null) : requestOptions.bodyContent(req, res, options);
+  var parseReqBody = (typeof options.parseReqBody === 'function') ? options.parseReqBody(req) : options.parseReqBody;
```

Comment:
I was thinking about a new flag, lets call it parseReqBodyFunction. It will be always function, and parseReqBody wil be always boolean, and of course, the will be mutually exclusive